src/server/dfly_bench.cc

```diff
@@ -741,17 +741,19 @@ void Driver::ParseRESP() {
   uint32_t consumed = 0;
   RedisParser::Result result = RedisParser::OK;
   RespVec parse_args;
+  constexpr string_view kMovedErrorKey = "MOVED"sv;
```

Comment:
are you sure this is correct? do we respond with MOVED and not -MOVED?

Reply:
We are comparing this now to parsed string so prefix '-' will be removed